pkg/gadgets/traceloop/tracer/tracer.go

```diff
@@ -432,8 +434,8 @@ func (t *Tracer) Read(containerID string) ([]*types.Event, error) {
 	// but they will be incomplete.
 	// One possible reason would be that the buffer is full and so it only remains
 	// some exit events and not the corresponding enter/
-	for _, enterTimestampEvents := range syscallEnterEventsMap {
-		for enterTimestamp, enterEvent := range enterTimestampEvents {
+	for enterTimestamp, enterTimestampEvents := range syscallEnterEventsMap {
+		for _, enterEvent := range enterTimestampEvents {
```

Comment:
FYI @eiffel-fl, I think the `enterTimestamp` variable is declared in the wrong `for` loop.

Reply:
Right, I am just focusing on the timestamps in this PR.